src/main/java/com/findmyrecycling/fmrenterprise/dao/FacilityDAOStub.java

```diff
@@ -39,19 +50,26 @@ public Facility fetchById(int id) {
         }
         return null;
     }
-
+    /**
+     * Deletes the facility with the id
+     * @param id
+     */
     @Override
     public void delete(int id) {
         int i = 0;
         for(Facility facility: facilities) {
             if (facility.getFacilityId() == id) {
                 facilities.remove(i);
-                break;
+                return;
```

Comment:
Are you sure you want return instead of break?  Mid method returns are often not recommended.  break allows the loop to continue to execute.